### DISCUSSION TITLE
Extra Features in Custom

### DISCUSSION THREAD

**Discussion by FlightDevil53** [flybywiresim/aircraft#4224] [Q&A] [answered]
Hello, how do I keep track of the extra custom features in the Experimental version. Is there a list, changelog or something alike?

Peter

**Comment by tracernz** [comment, marked as answer]
It doesn't have a changelog. The [readme has some information on what is done or not done](https://github.com/flybywiresim/a32nx/tree/autopilot/docs), the primarily the autoflight channel on discord and some of the others have info, or the [git commit history](https://github.com/flybywiresim/a32nx/commits/autopilot), but that may be hard to follow at times.

**Reply by FlightDevil53** [reply]
Wouldn't it be a great idea if someone kept track of all that is tested in Ecperimental. Now as a user it is almost impossible to see what is different from the Development version. That also makes it more difficult to create correct bug reports.
I just had a look at the readme.md, but it is not even clear for what version the readme is. It's in the a32nx/docs folder so it could apply for both development and experimental.

**Reply by tracernz** [reply]
> 
> 
> Wouldn't it be a great idea if someone kept track of all that is tested in Ecperimental. Now as a user it is almost impossible to see what is different from the Development version. That also makes it more difficult to create correct bug reports.
> I just had a look at the readme.md, but it is not even clear for what version the readme is. It's in the a32nx/docs folder so it could apply for both development and experimental.

The experimental version is exactly that, experimental. Once it's merged into the main master branch it will begin to be tracked in the way you desire. The link I gave you is to the readme.md in the autopilot branch, which is the experimental version.
